docs/source/api_implementation.rst

```diff
@@ -0,0 +1,23 @@
+Which API does it support?
+--------------------------
+============================= ===================== =================
+API name                      Implementation status Last checked date
+============================= ===================== =================
+`User provisioning API`_      OK                    2019-02-02
+`OCS Share API`_              Partially implemented 2019-02-02
+`WebDAV API`_                 Not implemented       2019-02-02
+`Activity app API`_           OK                    2019-02-02
+`Notifications app API`_      OK                    2019-02-02
```

Comment:
Please double-check this and the following rows - 2nd and 3rd column seem to be off one char.

Reply:
double checked it, it looks perfectly fine. 
![screenshot from 2019-02-06 00-31-07](https://user-images.githubusercontent.com/45518015/52308793-f304d180-29a6-11e9-9ed6-84f2fce9a3ff.png)

Reply:
Great, thanks!